src/tribler/core/utilities/simpledefs.py

```diff
@@ -6,29 +6,19 @@
 from enum import Enum
 from uuid import UUID
 
-DLSTATUS_ALLOCATING_DISKSPACE = 0  # TODO: make sure this get set when in this alloc mode
-DLSTATUS_WAITING4HASHCHECK = 1
-DLSTATUS_HASHCHECKING = 2
-DLSTATUS_DOWNLOADING = 3
-DLSTATUS_SEEDING = 4
-DLSTATUS_STOPPED = 5
-DLSTATUS_STOPPED_ON_ERROR = 6
-DLSTATUS_METADATA = 7
-DLSTATUS_CIRCUITS = 8
-DLSTATUS_EXIT_NODES = 9
-
-dlstatus_strings = [
-    'DLSTATUS_ALLOCATING_DISKSPACE',
-    'DLSTATUS_WAITING4HASHCHECK',
-    'DLSTATUS_HASHCHECKING',
-    'DLSTATUS_DOWNLOADING',
-    'DLSTATUS_SEEDING',
-    'DLSTATUS_STOPPED',
-    'DLSTATUS_STOPPED_ON_ERROR',
-    'DLSTATUS_METADATA',
-    'DLSTATUS_CIRCUITS',
-    'DLSTATUS_EXIT_NODES',
-]
+
+class Status(Enum):
+    DLSTATUS_ALLOCATING_DISKSPACE = 0
+    DLSTATUS_WAITING4HASHCHECK = 1
+    DLSTATUS_HASHCHECKING = 2
+    DLSTATUS_DOWNLOADING = 3
+    DLSTATUS_SEEDING = 4
+    DLSTATUS_STOPPED = 5
+    DLSTATUS_STOPPED_ON_ERROR = 6
+    DLSTATUS_METADATA = 7
+    DLSTATUS_CIRCUITS = 8
+    DLSTATUS_EXIT_NODES = 9
```

Comment:
The `Status` name looks too generic and unclear; I think it is better to use `DownloadStatus` instead. Also, the `DLSTATUS_` prefix before each name looks verbose; I think it is possible to remove it. As a result, instead of, for example, `Status.DLSTATUS_SEEDING`, we'll have `DownloadStatus.SEEDING`, which has the same length and is more descriptive.

```suggestion
class DownloadStatus(Enum):
    ALLOCATING_DISKSPACE = 0
    WAITING4HASHCHECK = 1
    HASHCHECKING = 2
    DOWNLOADING = 3
    SEEDING = 4
    STOPPED = 5
    STOPPED_ON_ERROR = 6
    METADATA = 7
    CIRCUITS = 8
    EXIT_NODES = 9
```